alerts/cluster-monitoring-operator/TelemeterClientFailures.md

```diff
@@ -0,0 +1,39 @@
+# TelemeterClientFailures
+
+## Meaning
+
+The alert `TelemeterClientFailures` is triggered when the Telemeter client
+fails to send metrics to the Telemeter server.
+
+The `telemeter-client-xxxxxxxxx-xxxxx` pod running in the `openshift-monitoring`
+namespace collects [selected platform metrics](https://docs.openshift.com/container-platform/latest/support/remote_health_monitoring/showing-data-collected-by-remote-health-monitoring.html#showing-data-collected-from-the-cluster_showing-data-collected-by-remote-health-monitoring)
+from the `prometheus-k8s` service at
+regular intervals(`4m30s`) using the `/federate` endpoint and ships the samples
+to the Telemetry endpoint using a custom protocol.
+
+## Impact
+
+When `telemeter-client` is having failures it will not be able to send metrics
+to the [telemetry system](https://docs.openshift.com/container-platform/latest/support/remote_health_monitoring/about-remote-health-monitoring.html#telemetry-about-telemetry_about-remote-health-monitoring).
+
+## Diagnosis
```

Comment:
We should probably break into 2 options:
* Telemeter client fails to pull metrics from Prometheus.
  * You'd need to check the availability of Prometheus.
* Telemeter client fails to push to the server.
  * It can be a server outage
  * It can be an issue with the cluster pull secret.